README.md

```diff
@@ -39,19 +95,6 @@ The GitHub issues are intended for bug reports and feature requests. For help an
 * Provides an easy-to-use programming model for working with transactions with Amazon QLDB.
```

Comment:
Seems like the Features section should be removed.

Reply:
Removed